<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="3" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-21T00:38:52 IST" name="api test suite" finished-at="2021-08-21T00:38:53 IST" duration-ms="440">
    <groups>
    </groups>
    <test started-at="2021-08-21T00:38:52 IST" name="/Users/abhilashsingh/IdeaProjects/weatherComparisonTest/src/test/java/tests/OpenweatherAPIPageTest.java" finished-at="2021-08-21T00:38:53 IST" duration-ms="440">
      <class name="tests.OpenweatherAPIPageTest">
        <test-method signature="checkStatusCode()[pri:0, instance:tests.OpenweatherAPIPageTest@5d800a56]" started-at="2021-08-21T00:38:52 IST" name="checkStatusCode" finished-at="2021-08-21T00:38:53 IST" duration-ms="434" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkStatusCode -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.OpenweatherAPIPageTest@5d800a56]" started-at="2021-08-21T00:38:53 IST" name="tearDown" finished-at="2021-08-21T00:38:53 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 5 out of bounds for length 5]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
at tests.OpenweatherAPIPageTest.tearDown(OpenweatherAPIPageTest.java:51)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at tests.TestNGRunner.main(TestNGRunner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.OpenweatherAPIPageTest -->
    </test> <!-- /Users/abhilashsingh/IdeaProjects/weatherComparisonTest/src/test/java/tests/OpenweatherAPIPageTest.java -->
  </suite> <!-- api test suite -->
  <suite started-at="2021-08-21T00:38:00 IST" name="ui test suite" finished-at="2021-08-21T00:38:47 IST" duration-ms="46846">
    <groups>
    </groups>
    <test started-at="2021-08-21T00:38:00 IST" name="/Users/abhilashsingh/IdeaProjects/weatherComparisonTest/src/test/java/tests/AccuweatherPageTest.java" finished-at="2021-08-21T00:38:47 IST" duration-ms="46846">
      <class name="tests.AccuweatherPageTest">
        <test-method is-config="true" signature="PageInitialisation()[pri:0, instance:tests.AccuweatherPageTest@636e4bf8]" started-at="2021-08-21T00:38:00 IST" name="PageInitialisation" finished-at="2021-08-21T00:38:12 IST" duration-ms="11890" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PageInitialisation -->
        <test-method signature="fetchAccuweatherData()[pri:0, instance:tests.AccuweatherPageTest@636e4bf8]" started-at="2021-08-21T00:38:12 IST" name="fetchAccuweatherData" finished-at="2021-08-21T00:38:47 IST" duration-ms="34838" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fetchAccuweatherData -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.AccuweatherPageTest@636e4bf8]" started-at="2021-08-21T00:38:47 IST" name="tearDown" finished-at="2021-08-21T00:38:47 IST" duration-ms="102" status="FAIL">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[Index 5 out of bounds for length 5]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 5
at tests.AccuweatherPageTest.tearDown(AccuweatherPageTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at tests.TestNGRunner.main(TestNGRunner.java:24)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.AccuweatherPageTest -->
    </test> <!-- /Users/abhilashsingh/IdeaProjects/weatherComparisonTest/src/test/java/tests/AccuweatherPageTest.java -->
  </suite> <!-- ui test suite -->
  <suite started-at="2021-08-21T00:38:53 IST" name="All Test Suite" finished-at="2021-08-21T00:38:53 IST" duration-ms="2">
    <groups>
    </groups>
    <test started-at="2021-08-21T00:38:53 IST" name="/Users/abhilashsingh/IdeaProjects/weatherComparisonTest/src/test/java/tests/CompareweatherParameters.java" finished-at="2021-08-21T00:38:53 IST" duration-ms="2">
      <class name="tests.CompareweatherParameters">
        <test-method signature="compare()[pri:0, instance:tests.CompareweatherParameters@25f7cc38]" started-at="2021-08-21T00:38:53 IST" name="compare" finished-at="2021-08-21T00:38:53 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- compare -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.CompareweatherParameters@25f7cc38]" started-at="2021-08-21T00:38:53 IST" name="tearDown" finished-at="2021-08-21T00:38:53 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.CompareweatherParameters -->
    </test> <!-- /Users/abhilashsingh/IdeaProjects/weatherComparisonTest/src/test/java/tests/CompareweatherParameters.java -->
  </suite> <!-- All Test Suite -->
</testng-results>
